plugins/modules/proxmox_storage_contents_info.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright Julian Vanden Broeck (@l00ptr) <julian.vandenbroeck at dalibo.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = """
+---
+module: proxmox_storage_contents_info
+short_description: List content from a Proxmox VE storage.
```

Comment:
Sorry, this is the only one without a full sentence :)
```suggestion
short_description: List content from a Proxmox VE storage
```